### PR TITLE
Feature #9661

### PR DESCRIPTION
**Pull request by kiokoman**
- [x] Redmine Issue: https://redmine.pfsense.org/issues/9661
- [x] Ready for review

### CODE REVIEW

File: src/usr/local/www/services_dhcp.php [hunk 1]
```diff
@@ -332,7 +332,7 @@ function validate_partial_mac_list($maclist) {
 		$input_errors[] = gettext("If a mac deny list is specified, it must contain only valid partial MAC addresses.");
 	}
 
-	if (($_POST['ntp1'] && (!is_ipaddrv4($_POST['ntp1']) && !is_hostname($_POST['ntp1']))) || ($_POST['ntp2'] && (!is_ipaddrv4($_POST['ntp2']) && !is_hostname($_POST['ntp2'])))) {
+	if (($_POST['ntp1'] && (!is_ipaddrv4($_POST['ntp1']) && !is_hostname($_POST['ntp1']))) || ($_POST['ntp2'] && (!is_ipaddrv4($_POST['ntp2']) && !is_hostname($_POST['ntp2']))) || ($_POST['ntp3'] && (!is_ipaddrv4($_POST['ntp3']) && !is_hostname($_POST['ntp3'])))) {
```
Comment (jim-p) [inline src/usr/local/www/services_dhcp.php]: Could you split this into multiple lines to make it easier to read? I know the old one was all one line but it was far too long as-is. Indent the line with four spaces after the leading tabs.

See [Indent Style](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style) for more info.

File: src/usr/local/www/services_dhcpv6.php [hunk 2]
```diff
@@ -328,7 +328,7 @@ function dhcpv6_apply_changes($dhcpdv6_enable_changed) {
 		}
 	}
 
-	if (($_POST['ntp1'] && !is_ipaddrv6($_POST['ntp1'])) || ($_POST['ntp2'] && !is_ipaddrv6($_POST['ntp2']))) {
+	if (($_POST['ntp1'] && !is_ipaddrv6($_POST['ntp1'])) || ($_POST['ntp2'] && !is_ipaddrv6($_POST['ntp2'])) || ($_POST['ntp3'] && !is_ipaddrv6($_POST['ntp3']))) {
```
Comment (jim-p) [inline src/usr/local/www/services_dhcpv6.php]: See previous comment about splitting this into multiple lines for readability.

**Comment by kiokoman** [comment]
sorry i'm new to all this stuff, is it good now?

**Comment by jim-p** [comment]
Yes, that looks great, thanks!

**Comment by pcmoore** [comment]
Thanks everyone!

**Comment by vktg** [comment]
works fine on 2.5.0.a.20200211.1811,
I can see three NTP servers in the packet capture

**Comment by rbgarga** [comment]
Manually merged.  Thanks!

**Pull request closed without merging**